app/controllers/pages_controller.rb

```diff
@@ -3,5 +3,7 @@ class PagesController < ApplicationController
 
   def home
     @projects = Project.all
+    # If current user has projects they are a sponsor
+    @sponsor = current_user.projects.count > 0
```

Comment:
should this be?:
if current_user.projects.count > 0
@sponsor = current_user
end